apps/src/templates/sectionAssessments/MultipleChoiceOverviewTable.jsx

```diff
@@ -8,47 +8,35 @@ import MultipleChoiceAnswerCell from './MultipleChoiceAnswerCell';
 
 export const COLUMNS = {
   QUESTION: 0,
-  ANSWER_1: 1,
-  ANSWER_2: 2,
-  NOT_ANSWERED: 3,
 };
 
-const answerOptionsOneFormatter = (percentAnsweredOptionOne, {rowData}) => {
-  return (
-      <MultipleChoiceAnswerCell
-        id={rowData.id}
-        percentValue={rowData.percentAnsweredOptionOne}
-        isCorrectAnswer={rowData.optionOneIsCorrect}
-      />
-  );
-};
+const alphabetMapper =  [
+                          commonMsg.answerOptionA(),
+                          commonMsg.answerOptionB(),
+                          commonMsg.answerOptionC(),
+                          commonMsg.answerOptionD(),
+                          commonMsg.answerOptionE(),
+                          commonMsg.answerOptionF(),
+                          commonMsg.answerOptionG(),
+                        ];
 
-const answerOptionsTwoFormatter = (percentAnsweredOptionOne, {rowData}) => {
+const answerColumnsFormatter = (percentAnswered, {rowData, columnIndex, rowIndex, property}, index) => {
+  const column = (columnIndex - 1);
+  const cell = rowData.answers[column];
```

Comment:
Nit: Might as well inline `column` since it's only used here: `const cell = rowData.answer[columnIndex - 1]`

Reply:
Fixed!  Thanks